cylc/flow/task_pool.py

```diff
@@ -741,6 +747,14 @@ def get_tasks_by_point(self):
 
         return point_itasks
 
+    def get_task(self, point, name):
+        """Retrieve a task from the pool."""
+        rel_id = f'{point}/{name}'
```

Comment:
Yeah, this was a little evil, this function's on me, I was trying to avoid another tokens call.

Reply:
Maybe `Tokens` needs to store the detokenised version instead of calling `detokenize(self)` on every property reference (so it only does it once, unless updated).

Reply:
I've been looking into caching `detokenise()` and/or the computed properties of `Tokens`, looks like the former is the most effective for increasing performance

Reply:
We're looking into caching in the Tokens module e.g. `@cached_property` or `@lru_cache` on `detokenise()` which would speed things up.

Reply:
👍   .. yeah computed properties might hurt if more is done than needed or no difference if it's one off use in some cases.

Reply:
Hard coded delimiter is a thing of the past, however, maybe a follow up once tokens are sorted